test/integration/targets/openssl_certificate/tasks/main.yml

```diff
@@ -71,4 +71,4 @@
 
     - import_tasks: ../tests/validate.yml
 
-  when: pyopenssl_version.stdout|version_compare('15.0.0', '>=')
+  when: pyopenssl_version.stdout|version_compare('0.15', '>=')
```

Comment:
@Spredzy - are you sure about that one? Seems like it fails on shippable: https://app.shippable.com/github/ansible/ansible/runs/36756/summary/console

Reply:
@MarkusTeufelberger Yep, pretty sure. The issue the CI is reporting is about `openssl_csr` module.

> 2017-09-11 15:53:29 TASK [openssl_csr : Generate CSR]
> 2017-09-11 15:53:30 fatal: [testhost]: FAILED! => {"changed": false, "failed": true, "msg": "You need to have PyOpenSSL>=0.15 to generate CSRs"}

Since 6819e71 is ensuring `pyOpenSSL>=0.15`, the same `block/when` condition should be added in the `openssl_csr/tasks/main.yml` file